### ISSUE TITLE
Is the config given the best ones?

### ISSUE THREAD

**Issue by Colezwhy** [jinhseo/OD-WSCL#9]
Hi jinhseo! Thank you for your wonderful job!
I am using the default configs provided by the repo, but during the training process, i found the loss goes to nan easily by the default settings.
Is that a normal phenomenon?
I am currently using 2*RTX3090(24G * 2)
Thank you!

**Comment by jinhseo** [comment]
Hi, regarding the nan loss issue, make sure that you maintain the batch size and learning rate provided in the config.
I recall nan loss when the learning rate is too high. However, when I used 2x RTX 3090 with a batch size of 8 and lr 0.01, I did not encounter any nan loss issue.

**Comment by Colezwhy** [comment]
> Hi, regarding the nan loss issue, make sure that you maintain the batch size and learning rate provided in the config. I recall nan loss when the learning rate is too high. However, when I used 2x RTX 3090 with a batch size of 8 and lr 0.01, I did not encounter any nan loss issue.

Thank you for replying, the issue is solved now. Wonderful job!
